app.rb

```diff
@@ -1,12 +1,25 @@
 require 'sinatra/base'
+# require 'sinatra/json'
 
 # Understands loading and managing pages appropriate to the website
 class PiratesBNB < Sinatra::Base
   set :method_override, true
 
-  get '/' do
+  get '/listings/new' do
```

Comment:
This method should be removed in the future. We will be handling this from Express.